src/main/webapp/dev/js/main/instructorFeedbackEdit.js

```diff
@@ -607,11 +607,18 @@ function formatNumberBox(participantType, questionNum) {
     const $questionForm = $(`#form_editquestion-${questionNum}`);
     const $numberOfEntitiesBox = $questionForm.find('.numberOfEntitiesElements');
 
-    if (participantType === 'STUDENTS' || participantType === 'TEAMS') {
+    if (participantType === 'STUDENTS' || participantType === 'TEAMS'
+            || participantType === 'INSTRUCTORS') {
         $numberOfEntitiesBox.show();
 
         const $numberOfEntitiesLabel = $numberOfEntitiesBox.find('.number-of-entities-inner-text');
-        $numberOfEntitiesLabel.html(participantType === 'STUDENTS' ? 'students' : 'teams');
+        if (participantType === 'STUDENTS') {
+            $numberOfEntitiesLabel.html('students');
+        } else if (participantType === 'TEAMS') {
+            $numberOfEntitiesLabel.html('teams');
+        } else {
+            $numberOfEntitiesLabel.html('instructors');
+        }
```

Comment:
This entire block can be `$numberOfEntitiesLabel.html(participantType.toLowerCase())`.